end_users/setup/README.md

```diff
@@ -2,9 +2,72 @@
 
 ## Table of Contents
 
+* [Robots](#robots)
+* [Webhook](#webhook)
+* [Hoist Configuration](#hoist-configuration)
+  * [Zuul Job Configuration](#zuul-job-configuration)
+  * [Zuul Layout Configuration](#zuul-layour-configuration)
 * [Merge Options](#merge-options)
   * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
 
+## Robots
+
+The first step to getting BonnyCI into your repository is to give the [robots](https://github.com/orgs/BonnyCI/teams/robots) team write access. This is so BonnyCI can automatically merge pull requests that pass all testing queues. Navigate to your repository's settings page, click on the `Collaborators & teams` section in the left-hand menu, and click `Add a team:` in the dropdown menu under the `Teams` section.
```

Comment:
Since this should change with the MVP, I'm not to concerned about linking to a team that the rest of GitHub can't see.

**NOTE:** I think that after we hit MVP, and this link is no longer needed, we should turn `no-dead-links` back on.